spec/vehicle_factory_spec.rb

```diff
@@ -0,0 +1,66 @@
+require './spec/spec_helper'
+
+RSpec.describe VehicleFactory do
+  before(:each) do
+    @factory = VehicleFactory.new
+    @dds = DmvDataService.new
+  end
+
+  it 'exists' do
+    expect(@factory).to be_an_instance_of(VehicleFactory)
+  end
+
+  it 'works with fake vehicles' do
+    cars = [
+      {vin: '123456789abcdefgh', year: 2012, make: 'Chevrolet', model: 'Cruz', engine: :ice},
+      {vin: '987654321abcdefgh', year: 2019, make: 'Chevrolet', model: 'Bolt', engine: :ev},
+      {vin: '1a2b3c4d5e6f', year: 1969, make: 'Chevrolet', model: 'Camaro', engine: :ice}
+    ]
+    vehicles = @factory.create_vehicles(cars)
+    first_vehicle = vehicles[0]
```

Comment:
Variable assignments might be unnecessary here - just test vehicle[0].vin, vehicle[1].year, etc